src/wh_server_crypto.c

```diff
@@ -245,7 +316,9 @@ static int hsmLoadKeyCurve25519(whServerContext* server, curve25519_key* key,
     int ret = 0;
     uint32_t privSz = CURVE25519_KEYSIZE;
     uint32_t pubSz = CURVE25519_KEYSIZE;
-    keyId |= WH_KEYTYPE_CRYPTO;
```

Comment:
Correct. Still not entirely sure I'm loving embedding the client id in the key id, since it was supposed to go into the access field instead.  Dunno.  Lemme think it over.